docs/001_develop/03_client-capabilities/021_desktop-interoperability/007_openfin-notifications-listener.mdx

```diff
@@ -0,0 +1,260 @@
+                >
+                </rapid-openfin-notification-listener>
+            );
+        }
+    ```
+</TabItem>
+
+<TabItem value="angular">
+    Declaration
+    ```html
+    <rapid-openfin-notification-listener></rapid-openfin-notification-listener>
+    ```
+
+    Usage
+    ```typescript
+    import { Component, CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+    @Component({
+        selector: 'my-root',
+        templateUrl: './my.component.html',
+        template: `
+            <rapid-openfin-notification-listener
+                #interopNotificationListenerRef
+                resource-name="ALL_NOTIFY_ALERT_RECORDS"
+            >
+            </rapid-openfin-notification-listener>
+        `,
+        standalone: true,
+        schemas: [CUSTOM_ELEMENTS_SCHEMA],
+    })
+    export class AppComponent {
+        @ViewChild('interopNotificationListenerRef') interopNotificationListener!: ElementRef;
+
+        ngAfterViewInit() {
+            if (this.interopNotificationListener.nativeElement) {
+                this.interopNotificationListener.nativeElement.buttons = [
+                    {
+                        task: 'warning-button',
+                        condition: ({details}) => details.NOTIFY_SEVERITY !== 'Warning',
+                        action: ({details, buttonElement, toastElement}) => {
+                            console.log({details, buttonElement, toastElement});
+                            toastElement.close();
+                        },
+                        appearance: 'secondary-orange',
+                        placeholder: 'Dismiss',
+                    },
+                ]
+
+                this.interopNotificationListener.nativeElement.notificationsPlatform = {
+                    id: 'your-platform-id',
+                    icon: 'http://url-to-your-icon.png',
+                    title: 'your platform title'
+                }
+            }
+        }
+    }
+    ```
+
+</TabItem>
+</Tabs>
+
+## DOM API
+
+Property and attribute binding examples for Genesis Component syntax. Closing tag omitted.
+
+### Attributes
+
+<table>
+    <thead>
+    <tr>
+        <th>Attribute</th>
+        <th>Type</th>
+        <th>Use</th>
+        <th>Example</th>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
+        <td>resource-name</td>
+        <td><code>string</code></td>
+        <td>The name of the server resource that pushes notifications to the client, normally 'ALL_NOTIFY_ALERT_RECORDS'</td>
+        <td>
+            ```typescript
+            <rapid-openfin-notification-listener resource-name="ALL_NOTIFY_ALERT_RECORDS">
+            ```
+        </td>
+    </tr>
+    </tbody>
+</table>
+
+### Properties
+
+<table>
+    <thead>
+    <tr>
+        <th>Property</th>
+        <th>Type</th>
+        <th>Use</th>
+        <th>Example</th>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
+        <td>toastButtons</td>
+        <td>
+            NotificationsPlatform <!-- To Do add link-->

```

Comment:
You've added the fdc3 docs right? So can't you do the link already?

Reply:
Link updated